packager/media/formats/mp4/mp4_media_parser.cc

```diff
@@ -510,6 +514,14 @@ bool MP4MediaParser::ParseMoov(BoxReader* reader) {
           codec_delay_ns =
               entry.dops.preskip * kNanosecondsPerSecond / sampling_frequency;
           break;
+        case FOURCC_mha1:
+        {
+          const AudioSampleEntry& entry = samp_descr.audio_entries[desc_idx];
```

Comment:
This line can be removed as `entry` is already assigned to the same value.

Reply:
Removed